secureli/modules/pii_scanner/pii_scanner.py

```diff
@@ -144,7 +144,10 @@ def _generate_scan_failures(
         for pii_found_file in pii_found_files:
             failures.append(
                 scan.ScanFailure(
-                    id="pii_scan", file=pii_found_file, repo=SECURELI_GITHUB
+                    id="pii_scan",
+                    file=pii_found_file,
+                    repo=SECURELI_GITHUB,
+                    exitCode="PII_SCAN_ISSUES_DETECTED",
```

Comment:
Question, why not use the ExitCode enum here? (might be misunderstanding how enums work in python)

Reply:
The enum will give you the value number of the exitCode.  So the goal is to defined the exitCode in the ScanFailure that matches the ENUM and when an Exit happens we can use, ExitCode[scanFailure.exitCode]

Reply:
Ok! Could we use the name attribute instead of the hardcoded string? 
<img width="811" alt="Screenshot 2024-05-15 at 1 41 46 PM" src="https://github.com/slalombuild/secureli/assets/85907688/87e1657c-0c10-4cdd-911e-503c6d18b6e0">
[Screenshot from enum doc page](https://docs.python.org/3/library/enum.html)

Reply:
Well at that point shouldn't we just pass the whole enum rather than just the name? Then when an exit happens we can just use the exitCode.value instead of using the name to lookup the value.